### ISSUE TITLE
Narrator does not read the state as ‘Expanded/Collapsed’ of "Variable options" button

### ISSUE THREAD

**Issue by MicrosoftIssueBot** [microsoft/calculator#1180]
**Repro Steps:** 
1. Launch "Calculator" app and turn on narrator. 
2. Home screen will be displayed.
3. Navigate to menu button and activate it.
4. Navigate to "Graphing" list item and activate it.
5. Navigate to 'Enter an expression" edit box any fill any equation.
6. Navigate to 'Variable options' button and activate it.
7. Observe the issue.

**Actual Result:**
Narrator stays silent on activating 'Variable options' button. Narrator does not read the state as ‘Expanded/Collapsed’ of "Variable options" button.

**Expected Result:**
Narrator should read the state as ‘Expanded/Collapsed’ of "Variable options" button on activate "Variable options" button.

**Comment by MicrosoftIssueBot** [comment]
This is your friendly Microsoft Issue Bot. I created this issue automatically as requested by a team member.
